Documentation/contributor-guide/roadmap.md

```diff
@@ -0,0 +1,42 @@
+# roadmap
+
+etcd uses GitHub milestones to track all tasks in each major or minor release. The roadmap.md file only records the most
+important tasks for each release.
+Please also refer to [here](https://docs.google.com/spreadsheets/d/1hFpIbkI0IDum0SPy8txj7bVHxQRrAB_82f1Yqz_Pa8M/edit#gid=0&fvid=1685003197)
+for more context. Note that etcd will continue to focus on technical debt over the next few major or minor releases.
+
+## v3.6.0
+| Title                                                              | Priority | Note                                                                                                         |
+|--------------------------------------------------------------------|----------|--------------------------------------------------------------------------------------------------------------|
+| Support downgrade                                                  | P0       | etcd will support downgrade starting from 3.6.0. But it will also support offline downgrade from 3.5 to 3.4. |
+| StoreV2 deprecation                                                | P0       | This task will be covered in both 3.6 and 3.7.                                                               |
+| Evaluate and (Gradulate or deprecate/remove) experimental features | P2       | This task will be covered in both 3.6 and 3.7.                                                               |
+| Support /livez and /readyz endpoints                               | P1       | It provides clearer APIs, and can also workaround the stalled writes issue                                   |
+| Bump gRPC                                                          | P0       | It isn't guaranteed to be resolved in 3.6, and might be postponed to 3.7 depending on the effort and risk.   |
+| Deprecate grpc-gateway or bump it                                  | P1       | It isn't guaranteed to be resolved in 3.6, and might be postponed to 3.7 depending on the effort and risk.   |
+| Release raft 3.6.0                                                 | P1       | etcd 3.6.0 will depends on raft 3.6.0                                                                        |
+| bbolt: Add logger into bbolt                                       | P1       | It's important to diagnose bbolt issues                                                                      |
+| bbolt: Add surgery commands                                        | P1       | Surgery commands are important for fixing corrupted db files                                                 |
+| Release bbolt 1.3.8                                                | P1       | etcd 3.6.0 will depends on bbolt 1.3.8                                                                       |
+
+## v3.7.0
```

Comment:
As above, consider linking through to the full milestone to save readers searching around to find it elsewhere.

Reply:
updated & thx